source/common/http/header_map_impl.cc

```diff
@@ -324,28 +319,24 @@ HeaderMapImpl::HeaderMapImpl(
 
 void HeaderMapImpl::updateSize(uint64_t from_size, uint64_t to_size) {
   // Removes from_size from cached_byte_size_ and adds to_size.
-  if (cached_byte_size_.has_value()) {
-    ASSERT(cached_byte_size_ >= from_size);
-    cached_byte_size_.value() -= from_size;
-    cached_byte_size_.value() += to_size;
-  }
+  ASSERT(cached_byte_size_ >= from_size);
+  cached_byte_size_ -= from_size;
+  cached_byte_size_ += to_size;
 }
 
 void HeaderMapImpl::addSize(uint64_t size) {
-  // Adds size to cached_byte_size_ if it exists.
-  if (cached_byte_size_.has_value()) {
-    cached_byte_size_.value() += size;
-  }
+  // Adds size to cached_byte_size_.
```

Comment:
nit: del obvious comment (I think it's fine to keep the method around for clarity vs. the other methods)

Reply:
Done.